src/kzg/mod.rs

```diff
@@ -17,6 +17,134 @@ struct TrustedSetup {
     g1_lagrange: Vec<String>,
 }
 
+#[derive(Clone, Debug)]
+pub struct KzgSettings {
+    pub roots_of_unity_brp: Box<[Fr; FIELD_ELEMENTS_PER_BLOB]>,
+    pub setup_g2_1: G2,
+    pub lagrange_setup_brp: Vec<G1Affine>,
```

Comment:
believe i missed this in #72 but probably best for consistency to also be a `Box`, or maybe both can be vectors

Reply:
good callout, given the value is now being passed in as a parameter i think that we can make them both vectors